src/azul/service/manifest_service.py

```diff
@@ -450,7 +450,9 @@ def get_cached_manifest_with_object_key(self,
                                                  authentication)
         current_source_key = generator.compute_source_key()
         manifest_key, source_key, extension = object_key.rsplit('/', 1)[-1].split('.')
-        if source_key != current_source_key:
+        # FIXME: Add support for long-lived API tokens
+        #        https://github.com/DataBiosphere/azul/issues/3328
```

Comment:
This comment should be added to an additional commit titled "Add FIXME (#3328)" (per our convention).